.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "sea-dsa"]
+  path = sea-dsa
+  url = https://github.com/seahorn/sea-dsa.git
+  branch = llvm-8.0
```

Comment:
Can we get a particular commit?
We'll have problems once they start updating sea-dsa.

Reply:
No, I think the way git submodules work is the version is fixed in the repo using them regardless of the updates in the submodule repos.